Calculator2/Calculator.cs

```diff
@@ -4,7 +4,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Calculator2
+namespace Calculator
 {
     public class Calculator
```

Comment:
Šis projektas (Calculatro.csproj) yra Console App tipo, jis būtinai turi turėti Main metodą. Šioje klasėje (Calculator) galite pridėti tuščią Main metodą, kad nemestų klaidos:
`static void Main() 
{

}`